### ISSUE TITLE
The URL prefix for library documentation pages differs from the old website.

### ISSUE THREAD

**Issue by ashtum** [boostorg/website-v2#728]
The Documentation URL shown on each library page has a prefix of `/doc/libs/boost_1_83_0` in comparison to `/doc/libs/1_83_0` in the old website.

```
https://www.boost.org/doc/libs/1_83_0/doc/html/boost_asio.html
https://www.boost.revsys.dev/doc/libs/boost_1_83_0/doc/html/boost_asio.html
```

**Comment by 4down** [comment]
I think this relates to what @vinniefalco had said during the meeting - these pages need to have the same URLs as they had at the old site.

What will need to happen to make them sync up?  (and, are there any versions in the past where the paths changed either just for that version or from that version onward, where we also need to accommodate for those differences?)  

@frankwiles @williln - but if there was a reason for changing them, maybe we do want to keep the new paths but set up redirects instead?  Or even if there wasn't a reason, would that simplify things?

**Comment by frankwiles** [comment]
Tracing through the code it turns out it already does this correctly.  See https://www.preview.boost.org/doc/libs/1_83_0/doc/html/boost_asio.html

We just have it in the other form in https://github.com/cppalliance/temp-site/blob/develop/libraries/models.py#L265-L270 so these URLs just need to be rewritten to remove the `boost_` prefix in the middle.

**Comment by williln** [comment]
@frankwiles @4down I'm not sure I'm following what the ask is. It sounds like the problem is that we're storing the docs data in S3 using directory names that include `boost_` but the old URLs for the docs don't use `boost_`. Is this a matter of adding a new pattern to https://github.com/cppalliance/temp-site/blob/develop/stage_static_config.json to allow us to interpolate docs urls that don't use `boost_`? 

Should we just change the directory names in S3 to conform to what we need?

**Comment by sdarwin** [comment]
>  the view smartly supports both formats

That means a critical aspect of the feature is already done. If they are both functional, it's not particularly important the format "boost_1_83_0" would cease to work.  It can probably stay as-is.  Rather what is important is that the URL string `/1_83_0/` is always shown, anywhere on the site.

The library details page `https://www.boost.io/libraries/asio/boost-1-83-0/` should display the link to docs as  `/1_83_0/`  : 
`https://www.boost.io/doc/libs/1_83_0/doc/html/boost_asio.html`

> Should we just change the directory names in S3 

Documentation is originating from published releases.   If you download and extract this file:  https://boostorg.jfrog.io/artifactory/main/release/1.84.0/source/boost_1_84_0.tar.gz

The resulting directory is `boost_1_84_0`, which is the format on wowbagger (the old website), and in S3, and it allows the deployment scripts to be able to simply extract and upload directories.   It's convenient to keep in place. Since "the view supports both formats" the new boost.io website isn't having trouble displaying directories, either way.  This favors keeping the extracted archive directories without changing them.

**Comment by kennethreitz** [comment]
I've been playing with updating the links on the site to point to the `/1_83_0/` format. Easier said than done :)

It is important to note that the `/1_83_0/` is supported, it's just not default at this time.

**Comment by kennethreitz** [comment]
Fixed by #1036

**Comment by kennethreitz** [comment]
This can now be seen on the staging site. Example: https://www.stage.boost.cppalliance.org/doc/libs/1_84_0/libs/algorithm/doc/html/index.html
